tests/toolkit/timeseries_tests.cc

```diff
@@ -0,0 +1,27 @@
+#include <gtest/gtest.h>
+
+#include "agent.h"
+
+#include "../test_context.h"
+#include "../test_agents/test_agent.h"
+
+namespace cyclus {
+namespace toolkit {
+
+TEST(TimeSeriesTests, Power) {
+  TestContext tc;
+  Agent* a = new TestAgent(tc.get());
+  RecordTimeSeries<POWER>(a, 42.0);
+  delete a;
+}
+
+TEST(TimeSeriesTests, RawPower) {
+  TestContext tc;
+  Agent* a = new TestAgent(tc.get());
+  RecordTimeSeries<double>("Power", a, 42.0);
+  delete a;
+}
```

Comment:
Are you looking at the output database separately to see that the information was actually recorded?